### DISCUSSION TITLE
missing Files from Example

### DISCUSSION THREAD

**Discussion by glen-02** [foxglove/community#836] [General]
in one of the examples i found
import { parse as parseMessageDefinition, parseRos2idl } from "@foxglove/rosmsg";

However i could not find @foxglove/rosmsg anywhere after installing. Is this normal?

**Comment by jtbandes** [comment]
You can install from npm: https://www.npmjs.com/package/@foxglove/rosmsg

**Comment by snosenzo** [comment]
also another thing to note is `parseRos2idl` has been moved to `@foxglove/ros2idl-parser`
